app/script/view_model/content/MessageListViewModel.js

```diff
@@ -137,8 +138,6 @@ z.viewModel.content.MessageListViewModel = class MessageListViewModel {
         this.conversation().isActiveParticipant() && this.conversation().inTeam() && this.conversation().isGuestRoom()
       );
     });
-
-    amplify.subscribe(z.event.WebApp.CONVERSATION.INPUT.CLICK, this.on_conversation_input_click.bind(this));
```

Comment:
What do you think about removing `z.event.WebApp.CONVERSATION.INPUT.CLICK` since we are not using it anymore?